.config/.flake8

```diff
@@ -0,0 +1,10 @@
+[flake8]
+max-line-length=88
+count=True
+ignore=E203,W503,E712
+extend-exclude=_version.py,lib,*_docs,geoips_dev_utils
+docstring-convention=numpy
+rst-roles=class,func,ref
+rst-directives=envvar,exception
+rst-substitutions=version
```

Comment:
Perhaps rename this back to .config/flake8 - since it doesn't really need to be hidden if it is not the "official" flake8 config (though I don't think it matters)

Reply:
I agree with Mindy here.